pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -389,6 +389,10 @@ type NetworkPolicyNodeStatus struct {
 	NodeName string `json:"nodeName,omitempty" protobuf:"bytes,1,opt,name=nodeName"`
 	// The generation realized by the Node.
 	Generation int64 `json:"generation,omitempty" protobuf:"varint,2,opt,name=generation"`
+	// The flag to mark the NetworkPolicy realization is failed on a Node.
+	RealizationFailure *bool `json:"realizationFailure,omitempty" protobuf:"bytes,3,opt,name=realizationFailure"`
+	// The error message for which the NetworkPolicy realization is failed on a Node
```

Comment:
The error message to describe why the NetworkPolicy realization is failed on the Node

Reply:
updated.